example/index.html

```diff
@@ -7,5 +7,9 @@
   <body>
 
     <script src="../dist/datefmt.js"></script>
+    <script>
+      let date = new Date(2013, 4, 8);
```

Comment:
Let's use `var` here since this won't be transpiled and wouldn't like people to have this fail.